src/stores/oneRMStore.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  IHistoryRecord,
+  ISet,
+  IWeight,
+} from "../types";
+
+type OneRMTriplet = {
+  max1RM?: IWeight;
+  max1RMHistoryRecord?: IHistoryRecord;
+  max1RMSet?: ISet;
+};
+
+const data = new Map<string, OneRMTriplet>();
+const subs = new Map<string, Set<(v: OneRMTriplet | undefined) => void>>();
```

Comment:
I frankly don't like this, sorry. Keeping state in global context is error-prone, and works poorly in tests.